cmd/vsphere-xcopy-volume-populator/internal/populator/secure-vmkfstools-wrapper.py

```
@@ -0,0 +1,311 @@
+                        logging.warning(f"Failed to remove RDM file {rdmfile}: {e}")
+        except FileNotFoundError:
+            pass
+        
+        # Remove the entire task directory
+        import shutil
+        shutil.rmtree(tmp_dir, ignore_errors=True)
+        
+        logging.info(f"Cleaned up task {task_id}")
+        print(XML_TEMPLATE.format(status="success", message="Cleanup completed"))
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)